juniper-vpn.py

```diff
@@ -22,10 +22,10 @@
 import socket
 import netifaces
 import datetime
+import re
 
 debug = False
 
-ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Because when written, the python module didn't support validation. If it's working now, the line can be removed.